Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM python:3.6.5-slim-stretch
 
+RUN apt-get update; apt-get -y install gnupg; apt-get -y install curl
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get update; apt-get -y install nodejs
```

Comment:
what version of nodejs are we running here?

Reply:
10.x